lib/checkPullRequestReview.js

```diff
@@ -36,9 +38,14 @@ const PTAL_COMMENTS = ['PTAL', 'Please take a look'];
  * @param {import('probot').Octokit.PullsGetReviewResponse} review
  */
 const handleChangesRequested = async (context, pullRequest, review) => {
+  let allLabels = pullRequest.labels.map((label) =>{
+    if (typeof label !== 'undefined') {
```

Comment:
This `if` statement is not required since label can not be `undefined`

Reply:
Will remove that.